docs/pages/guides/transactions.mdx

```diff
@@ -61,12 +61,12 @@ Use the [Transactions starter](https://github.com/framesjs/frames.js/tree/main/t
 
 The client will include the user's connected wallet address that will be executing the transaction in the frame action payload in the when a frame button with `action="tx"` set is pressed.
 
-The address is available in the context under the key `connectedAddress`.
+The address is available in the context under the key `address`.
 
 Note:
 
 - The address is only available when the user has connected a wallet to the client the frame button pressed is has a `tx` action.
-- `connectedAddress` differs from the `requesterVerifiedAddresses` returned by the [`farcasterHubContext`](/middleware/farcaster-hub-context) middleware.
+- `address` differs from the `requesterVerifiedAddresses` returned by the [`farcasterHubContext`](/middleware/farcaster-hub-context) middleware.
```

Comment:
breaking change?

Reply:
it is still available but marked as deprecated